tests/template/synergia_generate_test.py

```diff
@@ -8,7 +8,7 @@
 import pytest
 
 
-def test_generate_python():
+def xtest_generate_python():
```

Comment:
Prefer this: `@pytest.mark.skip(reason="no way of currently testing this")`

Reply:
b5c78fc96